capellambse/aird/parser/_filters/__init__.py

```diff
@@ -42,6 +41,8 @@
 GLOBAL_FILTERS: dict[str, GlobalFilter] = {}
 LOGGER = logging.getLogger(__name__)
 
+XP_PLUGIN = "/plugin/org.polarsys.capella.core.sirius.analysis/description/context.odesign#/"
```

Comment:
The prefix `XP_` suggests (and is used in other places for) an `etree.XPath` object. This however is not an XPath, it's just a simple "magic" string that we need to insert in the right place. (Call it platform path if you will, but that's still not an XPath.)